src/Components/App/App.js

```diff
@@ -24,9 +24,9 @@ class App extends Component {
   }
 
   showError() {
-    <article className={'error'}>
+    <article className="error">
       This movie is not currently available. Click below to confirm
-      <button onClick={() => this.showHome()} />
+      <button onClick={this.showHome} />
```

Comment:
Don't forget to add that callback function to the onClick - otherwise it will fire immediately

Reply:
https://reactjs.org/docs/faq-ajax.html
should componentDidMount() still have an api call within? I am having a hard time taking care of all error handling and state setting when importing request functions